test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,10 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
-});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    const names = [{name: 'Bart'}, {name: 'Lisa'}, {name:'Maggie'}];
+    expect(joinNames(names)).toBe('Bart, Lisa & Maggie');
+    const names2 = [{name: 'Bob'}, {name: 'Linda'}, {name: 'Tina'}, {name: 'Gene'}, {name: 'Louise'}];
+    expect(joinNames(names2)).toBe('Bob, Linda, Tina, Gene & Louise');
```

Comment:
Logic is good here, but its a bit cluttered and could be made much easier to read:
```
    const names = [{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }];
    const names2 = [
      { name: "Bob" },
      { name: "Linda" },
      { name: "Tina" },
      { name: "Gene" },
      { name: "Louise" },
    ];

    expect(joinNames(names)).toBe("Bart, Lisa & Maggie");
    expect(joinNames(names2)).toBe("Bob, Linda, Tina, Gene & Louise");
```

Its also a common practice to put all the setup at the top of the test and then all the assertions together at the bottom